code/datums/components/redirect_attack_hand_from_turf.dm

```diff
@@ -0,0 +1,116 @@
+/// Will redirect touching the turf it is on with your hand to the attack_hand of the parent object.
+/datum/component/redirect_attack_hand_from_turf
+	VAR_PRIVATE
```

Comment:
I hate you

Reply:
It's been merged before because it's the best way to do this